Prob2/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿public class Solution
+{
+    public static int LengthOfLongestSubstring(string s)
+    {
+
+        Dictionary<int, int> dict = new Dictionary<int, int>();
```

Comment:
@noor848  why do you use Dictionary here? why not List?

Reply:
both suggestions work also array